src/components/AssetsImagePreviewFilter/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckBox } from '@edx/paragon';
+
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import messages from './displayMessages';
+
+const AssetsImagePreviewFilter = ({ updateImagePreview, isImagePreviewEnabled }) => (
+  <CheckBox
+    // key={type.key} // I don't know if we need this. It's only for lists of elements, right?
+    // id={type.key} // what should this be?
+    name="imagePreviewEnabled" // what should this be?
+    label={<WrappedMessage message={messages.assetsImagePreviewFilterLabel} />}
+    checked={!isImagePreviewEnabled}
```

Comment:
I'm not sure how I feel about this. The checkbox "check" state and the actual meaning of that state are in conflict. If you check the checkbox, it disables the image preview, which necessitates the `!`. So, if the checkbox checked state is `true`, isImagePreviewEnabled is `false`. I wonder whether the checkbox label should be "Show Image Previews", and it should render checked by default. That could make the code cleaner, since a checkbox state of `true` would correspond to `true` for isImagePreviewEnabled. Just a thought. I don't think this code is terribly atrocious; it's just not super intuitive.